.eslintrc.cjs

```diff
@@ -44,7 +44,6 @@ module.exports = {
     "testing-library",
   ],
   rules: {
-    'react/prop-types': 'off',
```

Comment:
Return this back

Reply:
It was from upstream.
I didn't make any changes myself

Reply:
Your workflows failed